tests/aws/services/cloudformation/v2/test_change_set_parameters.py

```diff
@@ -0,0 +1,128 @@
+import pytest
+from localstack_snapshot.snapshots.transformer import RegexTransformer
+
+from localstack.services.cloudformation.v2.utils import is_v2_engine
+from localstack.testing.pytest import markers
+from localstack.utils.strings import long_uid
+
+
+@pytest.mark.skipif(condition=not is_v2_engine(), reason="Requires the V2 engine")
```

Comment:
A minor ease of use feature

```suggestion
@pytest.mark.skipif(condition=not is_v2_engine() and not is_aws_cloud(), reason="Requires the V2 engine")
```

Reply:
I changed this in other test files too